app/api/helpers/permissions.py

```diff
@@ -12,27 +12,34 @@ def second_order_decorator(inner_dec):
     :param inner_dec:
     :return:
     """
+
     def ddmain(outer_dec):
         def decwrapper(f):
             wrapped = inner_dec(outer_dec(f))
 
             def fwrapper(*args, **kwargs):
                 return wrapped(*args, **kwargs)
+
             return fwrapper
+
         return decwrapper
+
     return ddmain
 
 
 def jwt_required(fn, realm=None):
     """
     Modified from original jwt_required to comply with `flask-rest-jsonapi` decorator conventions
     View decorator that requires a valid JWT token to be present in the request
+    :param fn:
```

Comment:
Since you are adding the comment, can you also mention what this parameter means?